### ISSUE TITLE
Support other vendors via the Adoptium Marketplace

### ISSUE THREAD

**Issue by johanjanssen** [johanjanssen/TemurinChocolateyPackages#28]
Adoptium now supports Microsoft via the Marketplace.

https://adoptium.net/marketplace/
https://marketplace-api.adoptium.net/v1/assets/latestForVendors?feature_version=17&os=linux&architecture=x64&image_type=jdk&vendor=microsoft

**Comment by RedBaron2** [comment]
@johanjanssen 
This looks interesting.

**Comment by johanjanssen** [comment]
Yes indeed, from what I quickly saw the API seems quite similar, except for the url which now contains marketplace.

Unfortunately, they don't provide the Windows OpenJ9 artifacts from IBM yet.
